ext/xsl/xsltprocessor.c

```diff
@@ -760,33 +760,46 @@ PHP_METHOD(XSLTProcessor, registerPHPFunctions)
 {
 	zval *id = ZEND_THIS;
 	xsl_object *intern;
-	zval *array_value, *entry, new_string;
-	zend_string *name;
+	zval *zv = NULL, *entry, new_string;
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|z!", &zv) == FAILURE) {
```

Comment:
It would probably be better to use Z_PARAM_STR_OR_ARRAY_OR_NULL() here, to get standard behavior.

Reply:
So conversion to fast ZPP is ok then?

Reply:
Noted!